pages/overview/visualizations/cntrib_issue_assignment.py

```diff
@@ -0,0 +1,331 @@
+from dash import html, dcc, callback
+import dash
+import dash_bootstrap_components as dbc
+from dash.dependencies import Input, Output, State
+import plotly.graph_objects as go
+import pandas as pd
+import logging
+from dateutil.relativedelta import *  # type: ignore
+import plotly.express as px
+from pages.utils.graph_utils import get_graph_time_values, color_seq
+from queries.issue_assignee_query import issue_assignee_query as iaq
+import io
+from cache_manager.cache_manager import CacheManager as cm
+from pages.utils.job_utils import nodata_graph
+import time
+import datetime as dt
+
+PAGE = "overview"
+VIZ_ID = "cntrib_issue-assignment"
+
+gc_cntrib_issue_assignment = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H3(
+                    "Contributor Issue Assignment",
+                    className="card-title",
+                    style={"textAlign": "center"},
+                ),
+                dbc.Popover(
+                    [
+                        dbc.PopoverHeader("Graph Info:"),
+                        dbc.PopoverBody(
+                            """
+                            Visualizes the delta number of issue assignments for each contributor \n
+                            that meets the assignment criteria. This can help understand the workload'\n
+                            distribution between each contributor or maintainer over time.
+                            """
```

Comment:
Suggestion:

"Visualizes the number of issues assigned to distinct contributors over time."

I can see in the function "issue_assignment" that you calculate the delta, but I'm not sure if that's strictly relevant to the info section. My read of this visualization is "For each contributor that has at least one issue assigned to them, how many issues were assigned to them per time bin?"

I'm slightly apprehensive about suggesting an interpretation in the Info section. I think the interpretation provided is reasonable for most cases but I'd be more in favor of leaving it to the user to decide what the data means to them. Not a strong feeling though, if we decided to go that route I'd probably offer an insight like this on every visualization- imo, we're either impartial data visualizations providers or we aren't, and I don't have a value judgement either way.

Reply:
I personally think the delta is important because if people look for specifically the number of assignments over the time interval its not going to equal what we have 

Also why distinct? I feel like that is implied/not sure where this could be interpreted as non-distinct 

On providing context: personally I dont think data science is ever impartial, choosing what visualizations we provide inherently chooses to give a specific view on a community and each decision we make builds on that. Ill take it out though because I dont want to trigger a need to go through and update each info graph. I would say though we should consider in the future if we should because even if we dont put those things we will be repeating them ourselves to users when they come to us to help interpret.